Holoviz issue templates (#1)
holoviz/.github: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Suggest an idea for this project
title: ''
labels: TRIAGE
assignees: ''

---

```### FOLLOW THESE INSTRUCTIONS CAREFULLY

*ISSUES THAT DO NOT CONTAIN NECESSARY INFORMATION MAY BE CLOSED, IMMEDIATELY*

> The issue tracker is NOT the place for general support. For questions and technical assistance, come join the [Gitter](https://gitter.im/pyviz/pyviz).

Reviewer: ```suggestion
The issue tracker is NOT the place for general support. For usage questions and technical assistance, ask us on [Discourse](https://discourse.holoviz.org).
```
